core/util/src/main/java/org/arakhne/afc/progress/DefaultProgression.java

```diff
@@ -209,7 +209,7 @@ public double getPercent() {
 
 	@Override
 	public double getProgressionFactor() {
-		final double extent = this.max - this.min;
+		final double extent = this.max - (double) this.min;
```

Comment:
Not good because you force the usage of the double addition instead using the addition between integer before casting the resulting integer into a double.